appveyor.yml

```diff
@@ -30,6 +30,8 @@ before_build:
   - mkdir build
   - cd build
   - cmake .. -G "%CMAKE_GENERATOR_NAME%
+  - cmake .. -G "%CMAKE_GENERATOR_NAME% --target tests
+  - cmake .. -G "%CMAKE_GENERATOR_NAME% --target docs
```

Comment:
Have you tested that commit?

1. I find strange that it is in the `before_build:` section.
2. `--target` has no sense for CMake unless `--build <dir>` is also used.